java-manta-it/src/test/resources/testng-it.xml

```diff
@@ -11,78 +12,87 @@
         <parameter name="encryptionCipher" value="AES128/CTR/NoPadding"/>
         <groups>
             <run>
-                <exclude name="encryption-provider" />
+                <exclude name="expensive" />
             </run>
         </groups>
-        <packages>
-            <package name="com.joyent.manta.client.*">
-                <exclude name="com.joyent.manta.client.jobs"/>
-            </package>
-            <package name="com.joyent.manta.client.multipart.*"/>
-        </packages>
+        <classes>
+            <class name="com.joyent.manta.client.MantaClientIT" />
+            <class name="com.joyent.manta.client.MantaClientPutIT" />
+            <class name="com.joyent.manta.client.MantaClientMetadataIT" />
+            <class name="com.joyent.manta.client.MantaObjectOutputStreamIT" />
+
+            <!-- Range Operations only support AES/CTR -->
+            <class name="com.joyent.manta.client.MantaClientRangeIT" />
+            <class name="com.joyent.manta.client.MantaClientSeekableByteChannelIT" />
+
+            <!-- MPU only supports AES/CTR -->
+            <class name="com.joyent.manta.client.multipart.EncryptedServerSideMultipartManagerSerializationIT" />
+        </classes>
     </test>
     <test name="Manta Client Integration Tests [AES128/GCM Encrypted]">
-        <parameter name="usingEncryption" value="true"/>
         <parameter name="encryptionCipher" value="AES128/GCM/NoPadding"/>
-        <groups>
-            <run>
-                <exclude name="encryption-provider" />
-            </run>
-        </groups>
-        <packages>
-            <package name="com.joyent.manta.client.*">
-                <exclude name="com.joyent.manta.client.jobs"/>
-            </package>
-        </packages>
+        <classes>
+            <class name="com.joyent.manta.client.MantaClientIT" />
+            <class name="com.joyent.manta.client.MantaClientPutIT" />
+            <class name="com.joyent.manta.client.MantaClientMetadataIT" />
+            <class name="com.joyent.manta.client.MantaObjectOutputStreamIT" />
+        </classes>
     </test>
     <test name="Manta Client Integration Tests [AES128/CBC Encrypted]">
-        <parameter name="usingEncryption" value="true"/>
         <parameter name="encryptionCipher" value="AES128/CBC/PKCS5Padding"/>
-        <groups>
-            <run>
-                <exclude name="encryption-provider" />
-            </run>
-        </groups>
-        <packages>
-            <package name="com.joyent.manta.client.*">
-                <exclude name="com.joyent.manta.client.jobs"/>
-            </package>
-        </packages>
+        <classes>
+            <class name="com.joyent.manta.client.MantaClientIT" />
+            <class name="com.joyent.manta.client.MantaClientPutIT" />
+            <class name="com.joyent.manta.client.MantaClientMetadataIT" />
+            <class name="com.joyent.manta.client.MantaObjectOutputStreamIT" />
```

Comment:
You may want to comment out the line `<class name="com.joyent.manta.client.MantaObjectOutputStreamIT" />` for now because this test will fail due to #530.

Reply:
Done.